examples/Joints/Ingen/pinjoint.py

```diff
@@ -0,0 +1,16 @@
+from sympy import Symbol
+from pydy.bodies import Body
+from pydy.joints import PinJoint
+from pydy.joints_method import JointsMethod
+
+parent = Body('parent')
+child = Body('child')
+
+gravity = Symbol('gravity')
+child.add_force((0, child.get_mass() * gravity, 0), child.get_masscenter())
```

Comment:
If you add this to the child how do you know which direction to apply the force in the parent frame?

Let's avoid getter and setter methods. Anytime you have a get/set method you should simply make use of the `@property` decorator to control setting attributes. A method is also only needed if you need to check or change things on attribute change.